src/Reflection/ReflectionType.php

```diff
@@ -86,7 +86,7 @@ public function __toString()
                 return 'bool';
             case $this->type instanceof Types\Float_:
                 return 'float';
-            case $this->type instanceof Types\Void:
+            case $this->type instanceof Types\Void_:
```

Comment:
There is a test see [ReflectionTypeTest::testImplicitCastToString](https://github.com/Roave/BetterReflection/blob/master/test/unit/Reflection/ReflectionTypeTest.php#L45-L59), but types in phpDocumentor handles this failure case. The `default:` case below attempts to cast it to string, and types in phpDocumentor all implement `__toString()` so the test is testing the wrong thing. Therefore, it's pretty difficult to make this test fail unless we make a type that can't be cast to a string, so not sure what to suggest here :s